File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        printf("Height: ");
        height = get_int();
    } while(height < 0 || height > 23);
    
    for (int i = 1; i <= height; i++)
    {
        int j = height;
        while(j > i)
        {
            printf(" ");
            j--;
        }
        while(j > 0)
        {
            printf("#");
            j--;
        }
        printf("  ");
        while(j < i)
        {
            printf("#");
            j++;
        }
        printf("\n");
    }
}

File: mario.py
import cs50
while True:
    print("Height: ", end = "")
    height = cs50.get_int()
    if height > 0 and height < 23:
        break

for i in range(1, height + 1):
    j = height
    while j > i:
        print(" ", end = "")
        j -= 1
    while j > 0:
        print("#", end = "")
        j -= 1
    print("  ", end = "")
    while j < i:
        print("#", end = "")
        j += 1
    print()
